list_books.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
```

Comment:
Observations: The code does not follow modern PHP practices such as using namespaces and autoloading, which can lead to scalability issues as the application grows.

Recommendations:
- **Use Namespaces**: Organize your code using namespaces to avoid class name conflicts and improve code organization.
- **Autoloading**: Use Composer for autoloading classes, which simplifies the inclusion of files and dependencies.
- **Example**:
  ```php
  // In src/Repository/BookRepository.php
  namespace App\Repository;

  class BookRepository {
      // Class implementation
  }
  ```
  ```json
  // In composer.json
  {
      "autoload": {
          "psr-4": {
              "App\\": "src/"
          }
      }
  }
  ```
  ```bash
  composer dump-autoload
  ```